src/network/MissionControlProtocol.cpp

```diff
@@ -176,6 +177,31 @@ void MissionControlProtocol::handleJointPowerRequest(const json& j) {
 	}
 }
 
+void MissionControlProtocol::sendTelemetryData() {
+	auto heading = robot::readIMUHeading();
+	auto gps = robot::readGPS();
+	if (gps.isValid() && heading.isValid()) {
+		double orientX = 0.0;
+		double orientY = 0.0;
+		double orientZ = sin(heading.getData() / 2);
+		double orientW = cos(heading.getData() / 2);
+		double posX = gps.getData()[0];
+		double posY = gps.getData()[1];
+		double posZ = 1.0;
+		double recency = util::durationToSec(dataclock::now() - gps.getTime());
```

Comment:
Maybe take the older of the gps and imu timestamps? The recency should show the age of the oldest data in the packet.